init_pose_x: 0.0
init_pose_y: 0.0
init_pose_theta: 0.0

use_keyboard_control: false